src/index.tsx

```diff
@@ -60,6 +54,8 @@ import { AppRouter } from "components/organisms/AppRouter";
 
 import { LoadingPage } from "components/molecules/LoadingPage/LoadingPage";
 
+import { BugsnagErrorBoundary, addToBugsnagEventOnError } from "bugsnag";
```

Comment:
It should be at the top

Reply:
It shouldn't be on top. Please provide rationale why you think that's the case because I can only guess you've interpreted `"bugsnag"` as a 3rd party library import, while in fact it is import of a `./src/bugsnag/index.ts`.

The library imports being replaced are clearly identifiable by their `@scope/...`:
```ts
import Bugsnag from "@bugsnag/js";
import BugsnagPluginReact from "@bugsnag/plugin-react";
```